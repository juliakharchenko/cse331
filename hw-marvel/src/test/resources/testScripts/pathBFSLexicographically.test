# Tests finding a path between two nodes that have multiple edges of same length
# Shortest path between two nodes is decided by lexicographically least path.

LoadGraph nba nbaPlayers.csv

FindPath nba DWYANE-WADE LEBRON-JAMES