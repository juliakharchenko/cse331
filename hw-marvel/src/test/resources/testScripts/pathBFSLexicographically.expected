# Tests finding a path between two nodes that have multiple edges of same length
# Shortest path between two nodes is decided by lexicographically least path.

loaded graph nba

path from DWYANE-WADE to LEBRON-JAMES:
DWYANE-WADE to LEBRON-JAMES via CAVALIERS